.github/workflows/main.yml

```diff
@@ -84,7 +84,8 @@ jobs:
 
       - name: Create the conda environment
         shell: bash -l {0}
-        run: mamba install -q python=${{ matrix.python_version }} pip jupyterlab_pygments==0.1.0 pytest-cov pytest-rerunfailures nodejs=18 yarn ipywidgets matplotlib xeus-cling "traitlets>=5.0.3,<6" ipykernel
+        # TODO unpin pyzmq
```

Comment:
What's the issue with `pyzmq`?

Reply:
The `25.1.1` is not available yet on Mac OS

Reply:
Done

Reply:
I meant next to the TODO, or in a new issue.

Reply:
#1380